packages/realm/react-native/ios/RealmReact/RealmReact.mm

```diff
@@ -42,7 +42,6 @@ @interface RealmReact () <RCTBridgeModule, RCTInvalidating>
 @end
 
 @implementation RealmReact {
```

Comment:
I think the removed method declarations are still existing in the `RealmReact.h`

Reply:
Oh yeah - thanks!